code/game/objects/items/miscellaneous.dm

```diff
@@ -19,6 +19,62 @@
 	icon_state = "skub"
 	w_class = WEIGHT_CLASS_BULKY
 	attack_verb = list("skubbed")
+	
+/obj/item/herobeacon
+	name = "heroic beacon"
+	desc = "To summon heroes from the past to protect the future."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "gangtool-blue"
+	item_state = "radio"
+	var/static/list/display_names = list()
+
+/obj/item/herobeacon/attack_self(mob/user)
+	if(user.canUseTopic(src, NO_TK))
+		beacon_hero(user)
+
+/obj/item/herobeacon/proc/beacon_hero(mob/M)
+	if(!display_names.len)
+		var/static/list/herobox = typesof(/obj/item/storage/box/hero)
+		for(var/V in herobox)
+			var/atom/A = V
+			display_names[initial(A.name)] = A
+			
+	var/choice = input(M,"What heroic outfit would you like to order?","Historic Heroes") as null|anything in display_names
+	if(!choice || !M.canUseTopic(src, NO_TK) || !in_range(M, src) || M.restrained() || !M.canmove)
+		return
+	
+	var/poseheroicallyuntilthisworks = display_names[choice]
+	var/herobox = new poseheroicallyuntilthisworks()
+	M.put_in_hands(herobox)
+	qdel(src)
+
+/obj/item/storage/box/hero
+	name = "Courageous Tomb Raider - 1940's."
+	
+/obj/item/storage/box/hero/PopulateContents()
+	new /obj/item/clothing/head/fedora/curator(src)
+	new /obj/item/clothing/suit/curator(src)
+	new /obj/item/clothing/under/rank/curator/treasure_hunter(src)
+	new /obj/item/clothing/shoes/workboots/mining(src)
+	new /obj/item/melee/curator_whip(src)
+
+/obj/item/storage/box/hero/astronaut
+	name = "First Man on the Moon - 1960's."
+	
+/obj/item/storage/box/hero/astronaut/PopulateContents()
+	new /obj/item/clothing/suit/space/nasavoid(src)
+	new /obj/item/clothing/head/helmet/space/nasavoid(src)
+	new /obj/item/tank/internals/emergency_oxygen/double(src)
+	new /obj/item/gps(src)
```

Comment:
can you add a jetpack here

Reply:
Not in my design plan, they can use a baseball bat for propulsion, hack to take jetpacks or craft for additional bits. Im not adding it, sorry.